src/applets/show-desktop/ShowDesktopApplet.vala

```diff
@@ -77,8 +77,7 @@ public class ShowDesktopApplet : Budgie.Applet
         var window = Wnck.Window.@get(xid);
 
         if (window != null && window.is_minimized()) {
-            var utc_now = new DateTime.now_utc();
-            var now = (uint32) utc_now.to_unix();
+            uint32 now =  (uint32) Gtk.get_current_event_time();
```

Comment:
You don't need to cast `get_current_event_time` as a uint32, it already is. Can just do `window.unminimize(Gtk.get_current_event_time();`

Reply:
I'll rebase the commit and do your suggestion